services/com.mde-network.ep.toolfunctions.xtextfunction/src/test/java/com/mdenetnetwork/ep/toolfunctions/xtext/XtextToolTests.java

```diff
@@ -0,0 +1,82 @@
+package com.mdenetnetwork.ep.toolfunctions.xtext;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.ByteArrayOutputStream;
+import java.util.List;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+
+import com.mdenetnetwork.ep.toolfunctions.xtext.XtextTool;
+
+
+
+public class XtextToolTests {
```

Comment:
Generally, these tests need to be expanded to cover the key functionality.

Reply:
Agreed, added task to mdenet/educationplatform#81.